services/gallery-login.ts

```diff
@@ -17,12 +17,11 @@ export const GalleryLoginProvider = CredentialsProvider({
 
       const data = await response.json();
 
-      return {
-        id: data._id,
-        ...data,
-      };
+      return { id: data.id };
```

Comment:
The server isn't returning an id field, only _id or user_id (for individuals)/ gallery_id (for gallery)